### PR TITLE
MAT-8943: remove minimizealerts

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-8943](https://jira.cms.gov/browse/MAT-8943)
(Optional) Related Tickets:
https://github.com/MeasureAuthoringTool/madie-root/pull/318
https://github.com/MeasureAuthoringTool/madie-util/pull/145
https://github.com/MeasureAuthoringTool/madie-cql-library/pull/237/
https://github.com/MeasureAuthoringTool/madie-measure/pull/1106

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx [hunk 1]
```diff
@@ -344,7 +342,7 @@ export default function SavedCodesSubSection({
             </div>
           }
           canClose={false}
-          minimizeAlerts={featureFlags?.MinimizeAlerts}
+          minimizeAlerts={true}
```
Comment (jkotanchik-SB) [inline src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx]: See https://github.com/MeasureAuthoringTool/madie-cql-library/pull/237/files#r2310968415.

Comment (etan-sb) [reply src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx]: Confirmed that minimizeAlerts is defaulted to true by https://github.com/MeasureAuthoringTool/madie-design-system/pull/363

**Review by etan-sb — APPROVED** [review]
